README.md

```diff
@@ -117,6 +117,62 @@ To run the full test suite, simply run `rake` with no arguments.
 
 Rspec tests are used for unit testing the app, and some integration testing. Cucumber tests are used for testing API keys, javascript tests, and feature tests.
 
+## Linting
+
+`rake` will also run our linting: currently, only Rubocop.
+
+Rubocop checks for consistent style across the Ruby areas of the codebase.  We use a modified version of [Rubocop-Github](https://github.com/github/rubocop-github) to stay consistent with [SEC](https://github.com/EFForg/sec).
+
+I like to add Rubocop to a pre-commit hook, so that I can't commit non-compliant code. Here's a nice one:
```

Comment:
I don't think we should suggest this. It's a personal style whether or not every individual commit has to pass lint vs. the branch ultimately passing lint before it's merged. If we want to say anything about it then I think something like "please ensure your contributions lint correctly before opening a pull request" would be more appropriate.